tests/e2e/helper_test.go

```diff
@@ -38,15 +39,24 @@ import (
 )
 
 const (
-	servicemeshNamespace = "openshift-operators"
-	servicemeshOpName    = "servicemeshoperator"
-	serverlessOpName     = "serverless-operator"
-	ownedNamespaceNumber = 1 // set to 4 for RHOAI
-	deleteConfigMap      = "delete-configmap-name"
+	servicemeshNamespace     = "openshift-operators"
+	servicemeshOpName        = "servicemeshoperator"
+	serverlessOpName         = "serverless-operator"
+	ownedNamespaceNumber     = 1 // set to 4 for RHOAI
+	deleteConfigMap          = "delete-configmap-name"
+	operatorReadyTimeout     = 2 * time.Minute
+	componentReadyTimeout    = 4 * time.Minute // currently in component wait is set from 2-3 mins, use 4 mins to not fail test.
+	componentDeletionTimeout = 1 * time.Minute
+	crdReadyTimeout          = 1 * time.Minute
+	csvWaitTimeout           = 1 * time.Minute
+	dsciCreationTimeout      = 20 * time.Second // time required to get a DSCI in Ready state
+	dscCreationTimeout       = 20 * time.Second // time required to wait till DSC in Ready state
+	generalRetryInterval     = 10 * time.Second
+	generalWaitTimeout       = 2 * time.Minute
```

Comment:
if this timeout is used for DSC Ready, it's too small from what I get. I'm struggling with existing 5 minutes now and it looks like it is too close to the limit.

Reply:
`componentReadyTimeout    = 4 * time.Minute` is the one for the component to get Ready, and i put back the sleep time in the test before starting loop on component. 
XXXCreationTimeout is only for two CRs if they are created, not on component deployments